### DISCUSSION TITLE
[Docs] 为什么element-plus中能直接用vue2，有点奇怪，文档能出一element-plus+vue2版本的么

### DISCUSSION THREAD

**Discussion by yinwubikong** [element-plus/element-plus#17123] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

Vue Version: 3.2.47
Element Plus Version: 2.7.0
Browser / OS: Chrome/125.0.6422.142
Build Tool: Vite

### Additional comments

_(empty)_

**Comment by warmthsea** [comment]
这是 Vue 写法问题，你可以参考
https://cn.vuejs.org/guide/introduction.html#api-styles
